### ISSUE TITLE
Add all verses from search result to StudyPad

### ISSUE THREAD

**Issue by tuomas2** [AndBible/and-bible#2749]
> It would be great if I could put verses that pop up in search results into bookmarks/study maps right away, even multiple ones, without having to go to each single one.

https://support.andbible.org/scp/tickets.php?id=1931

**Comment by timbze** [comment]
Maybe this should be possible from "multiple references" window. Then this would be solved once #611 is implemented

**Comment by tuomas2** [comment]
I agree

**Comment by tuomas2** [comment]
It is already now possible because search results can be shown as multiple references window (there's item in 3 dot menu of search results)

**Comment by timbze** [comment]
> It is already now possible because search results can be shown as multiple references window (there's item in 3 dot menu of search results)

Oh wow and I didn't even know that! Wonder how long that feature has existed without me knowing... Does that mean #611 is solved?

**Comment by tuomas2** [comment]
I don't think it is really resolving #611. It was kind of "fast solution". Of course it's debatable if we want to implement #611 or not.
